relay-general/Cargo.toml

```diff
@@ -28,6 +28,7 @@ pest_derive = "2.1.0"
 regex = "1.5.5"
 relay-common = { path = "../relay-common" }
 relay-general-derive = { path = "derive" }
+relay-log = { path = "../relay-log" }
```

Comment:
Not sure if we want this dependency, but I want to log the error in `pii_config_uncached()` to reduce the noise, as opposed to every time `pii_config()` is called.